src/app/features/genes/components/gene-biodomains/gene-biodomains.component.scss

```diff
@@ -0,0 +1,65 @@
+#container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    height: 641px;
+
+    #left-hand-pane {
+        border: 1px solid #F1F3F5;
+        padding: 15px;
+
+        h3 {
+            text-align: center;
+        }
+    }
+
+    #right-hand-pane {
+        width: 525px;
+        background-color: #F1F3F5;
+        padding: 15px;
+        overflow: auto;
+    }
+}
+
+h3 {
+    font-size: 16px;
+    font-weight: bold;
+}
+
+ul {
+    li {
+        font-size: 14px;
+    }
+}
+
+
+
+// Custom scrollbar START
+
+/* WebKit-based browsers (Chrome, Safari) */
+::-webkit-scrollbar {
+        width: 6px;
+}
+
+::-webkit-scrollbar-track {
+background-color: transparent; /* Hide the track */
+}
+
+::-webkit-scrollbar-thumb {
+    background-color: #878E95;
+    border-radius: 5px;
+}
+
+::-webkit-scrollbar-thumb:hover {
+    background-color: #4A5056;
+}
+
+/* Firefox */
+@supports (scrollbar-color: #878E95 transparent) {
+    * {
+        scrollbar-color: #878E95 transparent;
+        scrollbar-width: thin;
+    }
+}
+
+// Custom scrollbar END
```

Comment:
Consider creating and importing a custom sass mixin, instead of just scoping to this component

Reply:
I agree. My plan is to rewrite the other charts to remove the dependence on d3 and revisit this after more details on the Visualization Framework get ironed out.